src/portal/src/i18n/lang/ko-kr-lang.json

```
@@ -0,0 +1,1943 @@
+        "SIGN_UP_REAL_NAME": "성과 이름",
+        "ITEM_REQUIRED": "필드(Field)를 입력하세요.",
+        "SCOPE_REQUIRED": "필드는 필수 항목이며 범위 형식이어야 합니다.",
+        "NUMBER_REQUIRED": "필드는 필수 항목이며 숫자 형식이어야 합니다.",
+        "PORT_REQUIRED": "필드는 필수 항목이며 유효한 포트 번호여야 합니다.",
+        "CRON_REQUIRED": "필드는 필수이며 cron 형식이어야 합니다.",
+        "EMAIL_EXISTING": "이메일주소가 이미 존재합니다.",
+        "USER_EXISTING": "이미 사용중인 사용자 이름입니다.",
+        "RULE_USER_EXISTING": "이미 사용중인 이름입니다",
+        "EMPTY": "이름을 입력하세요.",
+        "NONEMPTY": "공란은 허용하지 않습니다",
+        "REPO_TOOLTIP": "이 모드에서는 사용자가 이미지에 어떤 작업도 수행할 수 없습니다.",
+        "ENDPOINT_FORMAT": "엔드포인트는 반드시 HTTP:// 또는 HTTPS://로 시작해야합니다.",
+        "OIDC_ENDPOINT_FORMAT": "엔드포인트는 반드시 HTTPS://로 시작해야합니다.",
+        "OIDC_NAME": "OIDC 공급자의 이름입니다.",
+        "OIDC_ENDPOINT": "The URL of an OIDC-compliant server.",
+        "OIDC_SCOPE": "인증 시 OIDC 서버로 전송되는 범위입니다. “openid”와 “offline_access”를 포함해야 합니다. Google을 사용하는 경우 이 필드에서 'offline_access'를 삭제하세요.",
+        "OIDC_VERIFYCERT": "OIDC 서버가 자체 서명된 인증서를 통해 호스팅되는 경우 이 상자를 선택 취소하세요.",
+        "OIDC_AUTOONBOARD": "온보딩 화면을 건너뛰면 사용자가 사용자 이름을 변경할 수 없습니다. 사용자 이름은 ID 토큰에서 제공됩니다.",
+        "OIDC_USER_CLAIM": "사용자 이름을 검색하는 ID 토큰의 클레임 이름입니다. 지정하지 않으면 기본값은 'name'입니다.",
+        "NEW_SECRET": "시크릿(Secret)은 대문자 1개, 소문자 1개, 숫자 1개 이상이 포함된 8자 이상이여야합니다"
+    },
+    "PLACEHOLDER": {
+        "CURRENT_PWD": "현재 암호를 입력하세요",
+        "NEW_PWD": "새 암호를 입력하세요",
+        "CONFIRM_PWD": "새 암호를 확인합니다",
+        "USER_NAME": "사용자 이름(username)을 입력하세요",
+        "MAIL": "이메일 주소를 입력하세요",
+        "FULL_NAME": "성과 이름을 포함한 전체 이름을 입력하세요",
+        "SIGN_IN_NAME": "사용자 이름",
+        "SIGN_IN_PWD": "암호"
+    },
+    "PROFILE": {
+        "TITLE": "사용자 프로필",
+        "USER_NAME": "사용자 이름",
+        "EMAIL": "이메일(Email)",
+        "FULL_NAME": "성과 이름",
+        "COMMENT": "코맨트",
+        "PASSWORD": "암호",
+        "SAVE_SUCCESS": "사용자 프로필이 성공적으로 저장됐습니다..",
+        "ADMIN_RENAME_BUTTON": "사용자 이름 변경",
+        "ADMIN_RENAME_TIP": "사용자 이름을 \"admin@harbor.local\"로 변경하려면 버튼을 선택하세요. 이 작업은 취소할 수 없습니다.",
+        "RENAME_SUCCESS": "이름이 성공적으로 변경됐습니다!",
+        "RENAME_CONFIRM_INFO": "경고, 이름을 admin@harbor.local로 변경한 후에는 취소할 수 없습니다.",
+        "CLI_PASSWORD": "CLI 시크릿(Secret)",
+        "CLI_PASSWORD_TIP": "CLI 시크릿(Secret)Docker 또는 Helm 클라이언트의 비밀번호로 사용될 수 있습니다. OIDC 인증 모드가 활성화되면 로봇 계정을 사용하는 것이 좋습니다. CLI 암호는 ID 토큰의 유효성에 따라 달라지며 사용자가 정기적으로 UI에 로그인하여 토큰을 새로 고쳐야 하기 때문입니다.",
+        "COPY_SUCCESS": "복사 성공",
+        "COPY_ERROR": "복사 실패",
+        "ADMIN_CLI_SECRET_BUTTON": "SECRET 생성",
+        "ADMIN_CLI_SECRET_RESET_BUTTON": "시크릿(Secret) 업로드",
+        "NEW_SECRET": "시크릿(Secret)",
+        "CONFIRM_SECRET": "시크릿(Secret) 재입력",
+        "GENERATE_SUCCESS": "Cli 시크릿(Secret)이 성공적으로 설정됐습니다",
+        "GENERATE_ERROR": "Cli 시크릿(Secret) 설정에 실패했습니다",
+        "CONFIRM_TITLE_CLI_GENERATE": "시크릿(Secret)을 다시 생성할 수 있나요?",
+        "CONFIRM_BODY_CLI_GENERATE": "Cli 시크릿(Secret)을 재생성하면 이전 cli 시크릿이 삭제됩니다"
```

```suggestion
        "GENERATE_SUCCESS": "CLI 시크릿(Secret)이 성공적으로 설정됐습니다",
        "GENERATE_ERROR": "CLI 시크릿(Secret) 설정에 실패했습니다",
        "CONFIRM_TITLE_CLI_GENERATE": "시크릿(Secret)을 다시 생성할 수 있나요?",
        "CONFIRM_BODY_CLI_GENERATE": "CLI 시크릿(Secret)을 재생성하면 이전 cli 시크릿이 삭제됩니다"
```
CLI has to be all capital